package.json

```diff
@@ -24,7 +24,7 @@
   "main": "index.js",
   "scripts": {
     "start": "node example/index.js",
-    "test": "./node_modules/.bin/mocha"
+    "test": "./node_modules/.bin/mocha test/**/*.spec.js test/renderers/**/*.spec.js"
```

Comment:
Sorry, accidentally posted previous comment before finished writing it. Updated*